README.md

```diff
@@ -10,36 +10,72 @@ We welcome questions/comments/feedback (and pull requests).
 * IRC - [#calico](http://webchat.freenode.net?randomnick=1&channels=%23calico&uio=d4)
 * For Calico-on-Docker specific issues, please [raise issues](https://github.com/Metaswitch/calico-docker/issues/new) on Github.
 
-## Getting started
+## Getting Started
 
-The recommended way to get started with Calico for the first time is to follow the [Getting Started Guide](docs/GettingStarted.md).  You can be up and running within a matter of minutes.  The guide includes step by step instructions covering
-- setting up a 2 node Linux cluster either
-  - using Vagrant (on Virtual Box or VMware) for easy setup on your laptop or other virtual environment
-  - by manually setting up your own cluster
-- setting up Calico on the cluster
-- creating containers using Calico networking.
+To get started using [Calico Docker networking][calico-networking], we recommend running through one or more of the
+available tutorials described below.
 
-In addition to the [Getting Started Guide](docs/GettingStarted.md) we also have tutorials for:
-- [Calico on Amazon](docs/AWS.md)
-- [Calico on Google Compute Engine](docs/GCE.md)
-- [Calico on DigitalOcean](docs/DigitalOcean.md)
-- [Calico and Docker Swarm](docs/CalicoSwarm.md)
-- [Calico and Kubernetes](docs/kubernetes/README.md)
+If you would like to get involved writing code for calico-docker, or if you need to build binaries specific to your
+OS, checkout out the [Building and testing guide](docs/Building.md).
 
-Finally, if you want to try out networking containers using Calico with Docker's new [libnetwork network driver support](https://github.com/docker/libnetwork) then you can try out our [Ubuntu Vagrant libnetwork example](https://github.com/Metaswitch/calico-ubuntu-vagrant).
+### Tutorials
+
+Fully worked demonstrations of Calico Docker networking are available for the following environments:
+
+- [Vagrant install with CoreOS](docs/getting-started/VagrantCoreOS.md)
+- [Vagrant install with Ubuntu](docs/getting-started/VagrantUbuntu.md)
+- [Amazon Web Services (AWS)](docs/getting-started/AWS.md)
+- [Google Compute Engine (GCE)](docs/getting-started/GCE.md)
+- [DigitalOcean](docs/getting-started/DigitalOcean.md)
+
+For each of these environments, it is possible to run with one of three different 
+[networking options](#networking-options), described below.  Each of the demonstrations indicates where the instructions
+differ for each of these networking options.
+
+We also provide a demonstration of a [Docker Swarm cluster](docs/getting-started/powerstrip/CalicoSwarm.md) networked 
+using Calico. This particular demonstration utilizes the [powerstrip](#docker-with-powerstrip) networking option.
+
+For a quick start, we'd recommend following one of the Vagrant installations using the default docker networking.
+
+
+### Networking options
+
+#### Default docker networking
+
+This uses Dockers standard networking infrastructure, requiring you to explicitly add a created contained into a
+Calico network.

```

Comment:
I think it's worth mentioning that this is compatible with all docker version from 1.5 onwards.